lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -469,3 +469,18 @@ def test_svg_metadata():
     values = [node.text for node in
               rdf.findall(f'./{CCNS}Work/{DCNS}subject/{RDFNS}Bag/{RDFNS}li')]
     assert values == metadata['Keywords']
+
+
+@pytest.mark.parametrize('metadata,error,message', [
+    ({'Date': 1}, TypeError, "Invalid type for Date metadata. Expected str"),
+    ({'Date': [1]}, TypeError, "Invalid type for Date metadata. Expected it"),
```

Comment:
```suggestion
    ({'Date': [1]}, TypeError, "Invalid type for Date metadata. Expected iterable"),
```
to make it more readable. Break the line if necessary.